docs/.vitepress/theme/custom.css

```
@@ -0,0 +1,159 @@
+@font-face {
+  font-family: 'Geist';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)